src/contentmethods/methods.py

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# heritedcode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/heritedcode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+
+import requests as req
+
+base_content_swh_api_url = "https://archive.softwareheritage.org/api/1/content/"
+
+#this method is used to get information about content (aka a "blob") object. 
```

Comment:
This should be a docstring

Reply:
And BTW this is a function, not a method

Reply:
Okay sure, i'll update it.

Reply:
`"""
this function returns information like checksums, data_url, file_url, license_url
based on hash_type and hash as a json object
"""

def get_content_information(hash_type, hash):
    api = f"{base_swh_api_url}content/{hash_type}:{hash}/"
    response = requests.get(api)
    return response.json()`

This is the updated one.

Reply:
`"""
this function returns information like checksums, data_url, file_url, license_url
based on hash_type and hash as a json object
"""`